formula/src/main/java/io/reactivex/rxjava3/disposables/FormulaDisposableHelper.kt

```diff
@@ -0,0 +1,17 @@
+package io.reactivex.rxjava3.disposables
+
+import java.util.Objects
+
+/**
+ * Used due to issue with [Disposable.fromRunnable], please ignore outside of the internal library.
+ */
+object FormulaDisposableHelper {
```

Comment:
internal?

Reply:
If only. We can't, since we are pretending we are a part of the RxJava 3 package.

Reply:
Do you need to pretend, maybe move it into our package?

Reply:
Yep, we have to pretend, because `RunnableDisposable` is package private.

Reply:
I'm curious if it's enough to replace 
```kotlin
val disposable = Disposable.fromRunnable {
} 
```
with 
```kotlin
val runnable = Runnable {} 
val disposable = Disposable.fromRunnable(runnable)
```

Reply:
Good question. I don't think so, because `Disposible.fromRunnable` is a static interface method, which was added to Java 8 and therefore is likely the cause for this incompatibility with Firebase Test Lab